oslo_db/tests/sqlalchemy/test_exc_filters.py

```diff
@@ -821,6 +821,14 @@ def test_mysql_single(self):
             expected_value='2'
         )
 
+    def test_mysql_duplicate_entry_key_start_with_tablename(self):
+        self._run_dupe_constraint_test(
```

Comment:
Please enter doc string for this test case. What actually this test case is performing.

Reply:
done